src/components/ReportActionItem/MoneyRequestReportPreview/MoneyRequestReportPreviewContent.tsx

```diff
@@ -293,18 +281,17 @@ function MoneyRequestReportPreviewContent({
      * There is an edge case when there is only one distance expense with a pending route and amount = 0.
        In this case, we don't want to show the merchant or description because it says: "Pending route...", which is already displayed in the amount field.
      */
-    const {supportText} = useMemo(() => {
-        if (numberOfRequests === 1) {
-            return {
-                supportText: '',
-            };
-        }
-        return {
-            supportText: translate('iou.expenseCount', {
+    const expenseCount = useMemo(
+        () =>
+            translate('iou.expenseCount', {
                 count: numberOfRequests,
             }),
-        };
-    }, [translate, numberOfRequests]);
+        [translate, numberOfRequests],
+    );
+
+    const reportStatus = useMemo(() => getReportStatusTranslation(iouReport), [iouReport]);
```

Comment:
If we would only pass the state and status, we could also recompute this only when the relevant data changes not when anything on the report changes